datahub-web-react/vite.config.ts

```diff
@@ -67,6 +67,26 @@ export default defineConfig(({ mode }) => {
         // },
         envPrefix: 'REACT_APP_',
         build: {
+            rollupOptions: {
+                output: {
+                    manualChunks(id) {
```

Comment:
we probably should have some comments explaining why we added this

and specifically that the actual split is not important and can be adjusted in the future - only the final file sizes matter for us